src/logic/redux/config.js

```diff
@@ -0,0 +1,10 @@
+import storage from 'redux-persist/lib/storage';
+
+const storeConfig = {
+  key: 'root',
+  storage,
+  blacklist: [],
+  timeout: null,
```

Comment:
There is no such key for the store configuration in redux-persist, or is it? Have you read the docs?